src/Glpi/Asset/Capacity/HasDocumentsCapacity.php

```diff
@@ -68,12 +68,14 @@ public function getCloneRelations(): array
 
     public function isUsed(string $classname): bool
     {
+        // TODO filter using 'timeline_position' => 0 criteria
```

Comment:
This TODO have to be done before the merge.

Reply:
Isn't this incorrect/redundant? No link between a custom asset and document will be on an ITIL timeline, right?

Reply:
When an image is present in a rich text field, the corresponding document item has a `timeline_position` value that can be either `-1` (not set), or `1`, `2`, `3`, `4`. The `0` value correspond to a document that has been attached through the document tab, or a direct file upload field.

Filtering on the `0` value here will permit to not consider the capacity as used for a document that corresponds to a rich text image. There is already a filter on this value in the cleaning operation made when this capacity is disabled, to not break images inside rich text fields.